public/script.js

```diff
@@ -101,136 +98,124 @@ function makeBarChart(id, title, csv, annotations, showGroups) {
+                        scaleID: "x",
+                        type: "line",
+                        display: true,
+                        value: herdImmunityDate,
+                        borderColor: "#FFD700",
+                        borderWidth: 2,
+                        label: {
+                            content: "Est. " + new Date(herdImmunityDate).toLocaleDateString(),
+                            enabled: true
+                        }
+                    },
+                    {
+                        mode: "vertical",
+                        scaleID: "x",
+                        type: "line",
+                        display: true,
+                        value: latestDataDate,
+                        borderColor: "#03a5fc",
+                        borderWidth: 2,
+                        label: {
+                            content: "Latest data, " + new Date(latestDataDate).toLocaleDateString(),
+                            enabled: true
+                        }
+                    }
+                ]);
+        });
 }
 
 function makeLineChart(id, csv, annotations, limit) {
-    let dates = csv
-        .map(function(row) {
-            return row.real_date;
-        })
+    const dates = csv
+        .map(row => row.real_date)
         .filter(distinct);
-    let datasets = DOSES.map(function(dose) {
-        let vaccinated = csv
-            .filter(function(row) {
-                return row.dose == dose;
-            })
-            .map(function(row) {
-                let vaccinated = parseInt(row.vaccinated);
-                let population = parseFloat(row.population);
+    const datasets = DOSES.map(dose => {
+        const vaccinated = csv
+            .filter(row => row.dose === dose)
+            .map(row => {
+                const vaccinated = parseInt(row.vaccinated);
+                const population = parseFloat(row.population);
                 return {
                     x: row.real_date,
                     y: (vaccinated / population) * 100,
                     vaccinated: vaccinated,
-                    population: population,
+                    population: population
                 };
             });
         return {
             label: DOSE_LABELS[dose],
             backgroundColor: DOSE_COLORS[dose],
-            data: vaccinated,
+            data: vaccinated
         };
     });
 
     new Chart(id, {
         type: "line",
         data: {
             labels: dates,
-            datasets: datasets,
+            datasets: datasets
         },
         options: {
             tooltips: {
                 callbacks: {
-                    label: function(tooltipItem, data) {
-                        let dataset = data.datasets[tooltipItem.datasetIndex]
-                        let datasetData = dataset.data[tooltipItem.index];
+                    label: (tooltipItem, data) => {
+                        const dataset = data.datasets[tooltipItem.datasetIndex];
```

Comment:
Semicolons are entirely superfluous in JavaScript

Reply:
Keep the semicolons: https://stackoverflow.com/a/1169596

Reply:
Damn, that's rough. Maybe not entirely superfluous then